Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -11,12 +11,15 @@ import Dcrlibwallet
 
 class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDelegate {
     
-    @IBOutlet private var accountDropdown: DropMenuButton!
-    @IBOutlet private var imgWalletAddrQRCode: UIImageView!
-
-    @IBOutlet weak var subheader: UILabel!
-    @IBOutlet weak var lblWalletAddress: UILabel!
-    @IBOutlet var contentStackView: UIStackView!
+    var receiveAccountListView: ReceiveAccountListView!
+    
+    @IBOutlet weak var accountNameLab: UILabel!
+    @IBOutlet weak var walletLab: UILabel!
+    @IBOutlet weak var totalLab: UILabel!
+    @IBOutlet weak var walletAddressLabel: UILabel!
+    @IBOutlet weak var dropdownBtn: UIButton!
+    @IBOutlet weak var imgWalletAddrQRCode: UIImageView!
+    @IBOutlet weak var shareBtn: UIButton!
```

Comment:
This IBOutlet linking is not necessary. It's only usage is in the `setupExtraUI` function where a touchUpInside event handler was registered. This can/should be done from the inspector builder.